src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +21,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        int result = numbers.stream()
+                .map(n -> n.split(DELIMITER))
+                .flatMap(Arrays::stream)
+                .map(Integer::parseInt)
+                .reduce(Integer.MAX_VALUE, (a,b) -> b < a && pairingCheck(b) ? b : a);
+        if (result == Integer.MAX_VALUE) {
+            throw new RuntimeException("Can't get min value from list: " + numbers);
+        }
+        return result;
```

Comment:
Most obvious and readable solution is better in most cases
you reduce command is far from easy to be understood
```suggestion
        return result = numbers.stream()
                .map(numbersString -> numbersString.split(DELIMITER))
                .flatMap(Arrays::stream)
                .map(Integer::parseInt)
                .filter(i -> i % 2 == 0)
                .min()
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: " + numbers));
```